src/Symfony/Bridge/PhpUnit/SymfonyTestsListener.php

```diff
@@ -12,13 +12,15 @@
 namespace Symfony\Bridge\PhpUnit;
 
 use Doctrine\Common\Annotations\AnnotationRegistry;
+use PHPUnit\Framework\BaseTestListener;
+use PHPUnit\Framework\TestCase;
 
 /**
  * Collects and replays skipped tests.
  *
  * @author Nicolas Grekas <p@tchwork.com>
  */
-class SymfonyTestsListener extends \PHPUnit_Framework_BaseTestListener
+class SymfonyTestsListener extends BaseTestListener
```

Comment:
I think the bridge should not be updated to namespaces - only on master we should do that